go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/redhat-developer/app-services-sdk-go/kafkamgmt v0.4.0
 	github.com/redhat-developer/app-services-sdk-go/registryinstance v0.3.1
 	github.com/redhat-developer/app-services-sdk-go/registrymgmt v0.2.0
-	github.com/redhat-developer/service-binding-operator v0.8.0
+	github.com/redhat-developer/service-binding-operator v0.9.0
```

Comment:
Wasn't v0.10.0 announced? Maybe that could resolve it.

Reply:
0.10.0 is actually the version of the cluster that breaks things. 
0.10.0 also doesn't seem to compile https://github.com/redhat-developer/service-binding-operator/issues/1022

I have verified this for deployment config. So service binding is created and command works. 
but we cannot verify this end to end. I think we should merge this knowing that command works as it should